README.md

```diff
@@ -22,7 +22,7 @@ pipenv shell
 2. run the program through applying <br />
 
 ```
-python3 mqm_tool.py --folderPath [a absolute folder path] --maxDepth [maximum tree depth (default = 10)]
+python3 -m src.mqm --folderPath [a absolute folder path] --maxDepth [maximum tree depth (default = 10)]
```

Comment:
This is kind of an odd module to call...Can this not just be mqm and not src.mqm? With some restructuring?

Reply:
It could be if you are in the src folder. It is just odd because we have the repo set up for publishing to pypl, but still want it to be executable from the root. If you were to install this as a package you could just call `python3 -m mqm`.

Reply:
Mmm, I feel like ideally (and traditionally) the installation and all command line functionality for a module are called using the same name. This inconsistency is kind of confusing

Reply:
Yeah - I'd rather not have to call `src.mqm`; how can we fix that ? @nachtm

Reply:
After a lot of reading, I think the confusion surrounding this is actually just a documentation/understanding issue on my part, and is a function of how the `src` layout is supposed to work. The _advantage_ of this layout is that it is hard to use the local version without installing it -- we want to install to make sure that a) the code builds and b) the imports work for people who do install from PyPI for example. 

But, I think we can do both (run `python3 -m mqm` and only run the code as installed), and I've updated the pipfile to reflect that. Give it a shot and let me know if you run into problems.

Taking this into account, there will be two ways to run this code:
1. (eventually) `pip install` a release version from PyPI, and run `python -m mqm --args`. 
2. Clone this repository, activate the `pipenv shell`, and run `python -m mqm --args` from the root.

Is this intended? Do we need both? These are part of a larger discussion that I would like to have happen about the end goals for this project. If all we're doing is running a script, then I'm not entirely convinced we need to distribute through PyPI, and if we do distribute through PyPI but it's just an executable then I don't know if we need all of the scaffolding we have.

Reply:
I was able to get this thing up and running - thanks for making the updates